composer.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "wmde/ContentStaging",
+	"description": "",
+	"require-dev": {
+		"squizlabs/php_codesniffer": "~2.1",
+		"phpmd/phpmd": "~2.1"
+	},
+	"scripts": {
+		"cs": [
+			"vendor/bin/phpcs ./*Content* --standard=phpcs.xml --extensions=php -sp",
+			"vendor/bin/phpmd ./*Content* text phpmd.xml"
```

Comment:
thanks did not realize that since phpmd only runs when phpcs finishes without errors
fixed and changed phpcs to only check files in local directory